system_test/aest_perf_SUITE.erl

```diff
@@ -182,8 +182,11 @@ wait_for_sync(Nodes, Height, Cfg) ->
     SyncTimeout = proplists:get_value(sync_timeout, Cfg),
     wait_for_value({height, Height}, Nodes, SyncTimeout * Height, Cfg).
 
-get_block(NodeName, Height, Cfg) ->
-    request(NodeName, [v2, 'block-by-height'], #{height => Height}, Cfg).
+get_block(NodeName, Height) ->
+    case request(NodeName, 'GetBlockByHeight', #{height => Height}) of
+        {ok, 200, Block} -> Block;
+        _ -> error(undefined)
```

Comment:
Can this raise some more error information to make the test cases easier to debug? Something like `Error -> error({error_getting_block, NodeName, Height, Error})`